src/server/fetch.ts

```diff
@@ -352,7 +352,7 @@ export class GlobalFetchRequest extends FetchRequest {
     }
     this._options = {
       baseURL: options.baseURL,
-      userAgent: options.userAgent || '',
+      userAgent: options.userAgent || 'Playwright',
```

Comment:
nit: to be more compliant we should maybe do Playwright/v1.15.0 instead: https://datatracker.ietf.org/doc/html/rfc7231#section-5.5.3

Since WAFs parse them and might find just Playwright too little, I would follow what the other http clients also do.

Reply:
Added version.